src/components/UserForm.js

```diff
@@ -124,6 +125,20 @@ const UserForm = ({ variant, standalone = true, onSubmitFulfilled }) => {
           onChange={handleChange("username")}
           InputLabelProps={{ shrink: true }}
         />
+        <TextField
```

Comment:
It should only be displayed when variant === "signup"

Reply:
@olavea
I removed email from log in form, looks like it is working,
😸 👍

Reply:
```
       {variant === "signup" ? (
          <TextField
            variant="outlined"
            margin="normal"
            required
            fullWidth
            id="email"
            label="Email"
            name="email"
            placeholder="unicorn@usepow.app"
            value={state.email}
            autoComplete="email"
            onChange={handleChange("email")}
            InputLabelProps={{ shrink: true }}
          />
        ) : (
          "current-email"
        )}
 
```

Reply:
I learned a new thing:

If a customer 🎩 is signing UP ⬆️ she can write her email:  

```
{variant === "signup" ? (
  <TextField
  ...
    label="Email"
  ...
    />
  ) : (
    "current-email"
)}
```

Else a customer 🎩 can not write her email.

#usepowapp #100daysofGatsby #webdevnewbie